whitepapers/0013_ccq.md

```diff
@@ -309,50 +344,82 @@ Currently the only supported query types are `eth_call` and `eth_call_by_timesta
   []byte     per_chain_responses
   ```
-- Per-Chain Responses
 
-  ```go
-  u16     chain_id
-  u8      type
-  uint32  response_len
-  []byte  response
-  ```
+### Per-Chain Responses
+
+All per-chain responses begin with following header.
+
+```go
+u16     chain_id
+u8      type
+uint32  response_len
+[]byte  response
+```
+
+#### EVM Query Responses
+
+1. eth_call (query type 1) Response Body
+
+   ```go
+   u64         block_number
+   [32]byte    block_hash
+   u64         block_time_us
+   u8          num_results
+   []byte      results
+   ```
+
+   ```go
+   u32         result_len
+   []byte      result
+   ```
+
+2. eth_call_by_timestamp (query type 2) Response Body
+
+   ```go
+   u64         target_block_number
+   [32]byte    target_block_hash
+   u64         target_block_time_us
+   u64         following_block_number
+   [32]byte    following_block_hash
+   u64         following_block_time_us
+   u8          num_results
+   []byte      results
+   ```
+
+   ```go
+   u32         result_len
+   []byte      result
+   ```
+
+3. eth_call_with_finality (query type 3) Response Body
+   The response for `eth_call_with_finality` is the same as the response for `eth_call`, although the query type will be three instead of one.
+
+#### Solana Query Responses
+
+1. sol_account (query type 10) Response Body
+
+   ```go
+   u64         slot_number
+   u64         block_time_us
+   u8          num_results
+   []byte      results
+   ```
+
+   - The `slot_number` is the slot number returned by the query.
+   - The `block_time_us` is the timestamp of the block associated with the slot.
+   - The array of results returns the data for each account queried.
+
+   ```go
+   u64         lamports
+   u32         owner_len
+   []byte      owner
+   u32         result_len
+   []byte      result
+   ```
 
-  1. eth_call (query type 1) Response Body
-
-     ```go
-     u64         block_number
-     [32]byte    block_hash
-     u64         block_time_us
-     u8          num_results
-     []byte      results
-     ```
-
-     ```go
-     u32         result_len
-     []byte      result
-     ```
-
-  2. eth_call_by_timestamp (query type 2) Response Body
-
-     ```go
-     u64         target_block_number
-     [32]byte    target_block_hash
-     u64         target_block_time_us
-     u64         following_block_number
-     [32]byte    following_block_hash
-     u64         following_block_time_us
-     u8          num_results
-     []byte      results
-     ```
-
-     ```go
-     u32         result_len
-     []byte      result
-     ```
-
-  3. eth_call_with_finality (query type 3) Response Body
-     The response for `eth_call_with_finality` is the same as the response for `eth_call`, although the query type will be three instead of one.
+   - The `lamports` is the number of lamports assigned to the account.
+   - The `owner` is the public key of the owner of the account.
+   - The `result` is the data returned by the account query.
```

Comment:
I do wonder if anyone will have a use case for lamports or the other fields (executable, rentEpoch, size), though I'd agree those three seem less likely than lamports. More opinions might be good though. At worst, to include them is one bool and two u64s

Reply:
Added `rent_epoch` and `executable`. I don't see a `size` returned by the query.

Reply:
The Go API does not return the size / space. Had an offline discussion, and decided we could do without it now.